automation_tools/__init__.py

```diff
@@ -1622,6 +1623,30 @@ def setenforce(mode):
     run('setenforce {0}'.format(mode))
 
 
+def host_pings(host):
```

Comment:
I am seriously not understanding how do I use timeout here. Because I have to loop the ping until the status code is 0/1.